File: content/LLD/Reference-Code/21. Ride Sharing Project - Bad Example/driver.py
from location import Location
from vehicle import Vehicle


class Driver:
    def __init__(self, name, location: Location, vehicle: Vehicle):
        self.name: str = name
        self.location: Location = location
        self.vehicle: Vehicle = vehicle

    def get_location(self) -> Location:
        return self.location

    def set_location(self, location: Location) -> None:
        self.location = location

File: content/LLD/Reference-Code/21. Ride Sharing Project - Bad Example/Driver.java
public class Driver {
    public String name;
    public Location location;
    public Vehicle vehicle;

    public Driver(String name, Location location, Vehicle vehicle) {
        this.name = name;
        this.location = location;
        this.vehicle = vehicle;
    }

    public Location getLocation() {
        return this.location;
    }

    public void setLocation(Location location) {
        this.location = location;
    }
}
